python/foglamp/services/core/api/scheduler.py

```diff
@@ -278,12 +289,73 @@ def _extract_args(data, curr_value):
         schedule = {
             'id': str(sch.schedule_id),
             'name': sch.name,
-            'processName': sch.process_name,
+            "processName": sch.process_name,
             'type': Schedule.Type(int(sch.schedule_type)).name,
             'repeat': sch.repeat.total_seconds() if sch.repeat else 0,
             'time': (sch.time.hour * 60 * 60 + sch.time.minute * 60 + sch.time.second) if sch.time else 0,
             'day': sch.day,
-            'exclusive': sch.exclusive
+            'exclusive': sch.exclusive,
+            'enabled': sch.enabled
+        }
+
+        return web.json_response(schedule)
+    except (ValueError, ScheduleNotFoundError) as ex:
+        raise web.HTTPNotFound(reason=str(ex))
+
+
+async def enable_schedule(request):
+    """
+    Enable the given schedule from schedules table
+
+    :Example: curl -X PUT  http://localhost:8081/foglamp/schedule/enable/ac6dd55d-f55d-44f7-8741-984604bf2384
+    """
+
+    try:
+        schedule_id = request.match_info.get('schedule_id', None)
+
+        if not schedule_id:
+            raise web.HTTPBadRequest(reason='Schedule ID is required.')
+
+        try:
+            assert uuid.UUID(schedule_id)
+        except ValueError as ex:
+            raise web.HTTPNotFound(reason="Invalid Schedule ID {}".format(schedule_id))
+
+        status = await server.Server.scheduler.enable_schedule(uuid.UUID(schedule_id))
+
+        schedule = {
+            'scheduleId': schedule_id,
+            'status': status
+        }
+
+        return web.json_response(schedule)
+    except (ValueError, ScheduleNotFoundError) as ex:
+        raise web.HTTPNotFound(reason=str(ex))
+
+
+async def disable_schedule(request):
+    """
+    Disable the given schedule from schedules table
+
+    :Example: curl -X PUT  http://localhost:8081/foglamp/schedule/disable/ac6dd55d-f55d-44f7-8741-984604bf2384
```

Comment:
Right URL is `curl -X PUT  http://localhost:8081/foglamp/schedule/ac6dd55d-f55d-44f7-8741-984604bf2384/disable`